cypress/platform/bundle-test.js

```diff
@@ -1,4 +1,4 @@
-import mermaid from '../../dist/mermaid.core';
+import mermaid from '../../dist/mermaid';
```

Comment:
@sidharthv96 Are we removing mermaid.core? If so this would be breaking change down streams. We should describe supported ways of using mermaid in the docs and setup corresponding tests in our release flow. Ideally we keep the API the same.

Reply:
<img width="332" alt="image" src="https://user-images.githubusercontent.com/10703445/189322594-34a3f448-a21d-4c4d-b004-f24d4a408d81.png">

Mermaid core seemed to be a development build. We can keep a copy with `mermaid.core.js` for backwards compatibility.

Reply:
Yes let's do that!

Reply:
It looks like `mermaid.core.js` also doesn't bundle anything in `node_modules/`.

https://github.com/mermaid-js/mermaid/blob/d78adc6fb94cac4af2068af49b027514e2d43293/.webpack/webpack.config.babel.js#L11

We should be able to copy this in `esbuild` by using `external: "./node_modules/*"`, see https://esbuild.github.io/api/#external